ember-cli-build.js

```diff
@@ -70,12 +73,42 @@ module.exports = function() {
   var libTree = find(jsTree, {
     include: ['*/index.js', '*/lib/**/*.js']
   });
+
   libTree = merge([libTree, HTMLTokenizer, handlebarsInlinedTrees.compiler]);
 
   var es6LibTree = mv(libTree, 'es6');
+
+  /*
+   * ES5 Named AMD Build
+   */
   libTree = transpile(libTree, 'ES5 Lib Tree');
   var es5LibTree = mv(libTree, 'named-amd');
 
+  /*
+   * CommonJS Build
+   */
+  tsOptions.tsconfig.compilerOptions.module = "commonjs";
```

Comment:
Are these objects being deeply cloned when we pass them to `typescript`?  I am concerned that sharing the object by ref here would lead to commonjs being used across the board...